msctl

```diff
@@ -2186,46 +2326,50 @@ fi
 if [ ! -s "$MSCS_DEFAULTS" ]; then
   mscs_defaults >$MSCS_DEFAULTS
 fi
+
 # Default values in the script can be overridden by adding certain key/value
 # pairs to one of the mscs.defaults files mentioned above. Default values in
 # the script will be used unless overridden in one these files.
 #
 # The following keys are available:
-#   mscs-location                - Location of the mscs files.
-#   mscs-worlds-location         - Location of world files.
-#   mscs-versions-url            - URL to download the version_manifest.json file.
-#   mscs-versions-json           - Location of the version_manifest.json file.
-#   mscs-versions-duration       - Duration (in minutes) to keep the version_manifest.json file before updating.
-#   mscs-lockfile-duration       - Duration (in minutes) to keep lock files before removing.
-#   mscs-default-world           - Default world name.
-#   mscs-default-port            - Default Port.
-#   mscs-default-ip              - Default IP address.
-#   mscs-default-version-type    - Default version type (release or snapshot).
-#   mscs-default-client-version  - Default version of the client software.
-#   mscs-default-client-jar      - Default .jar file for the client software.
-#   mscs-default-client-url      - Default download URL for the client software.
-#   mscs-default-client-location - Default location of the client .jar file.
-#   mscs-default-server-version  - Default version of the server software.
-#   mscs-default-jvm-args        - Default arguments for the JVM.
-#   mscs-default-server-jar      - Default .jar file for the server software.
-#   mscs-default-server-url      - Default download URL for the server software.
-#   mscs-default-server-args     - Default arguments for a world server.
-#   mscs-default-initial-memory  - Default initial amount of memory for a world server.
-#   mscs-default-maximum-memory  - Default maximum amount of memory for a world server.
-#   mscs-default-server-location - Default location of the server .jar file.
-#   mscs-default-server-command  - Default command to run for a world server.
-#   mscs-backup-location         - Location to store backup files.
-#   mscs-backup-log              - Lcation of the backup log file.
-#   mscs-backup-excluded-files   - Comma separated list of files and directories excluded from backups.
-#   mscs-backup-duration         - Length in days that backups survive.
-#   mscs-log-duration            - Length in days that logs survive.
-#   mscs-detailed-listing        - Properties to return for detailed listings.
```

Comment:
This was also moved here in an earlier PR so it matches the default value listing section below

Reply:
got it, fixed. thank you